index.html

```diff
@@ -3,25 +3,164 @@
 
     <head>
         <meta charset="UTF-8">
-        <title>Mobile Portolio Setup</title>
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href='https://fonts.googleapis.com/css?family=Playfair Display' rel='stylesheet'>
+        <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+        <link href="https://fonts.googleapis.com/css2?family=Space+Mono&display=swap" rel="stylesheet">
         <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
         <meta name="viewport" content="width=device-width, initial-scale=1">
         <link rel="stylesheet" href="styles.css">
+        <title>Building Portfolio(Mobile)</title>
     </head>
 
   <body>
-          <header>
-            <nav class="Nav-all"> 
-        <ul>
-            <li><img class="image" src="/img/image_geometry_1-1.png"></li>
-            <li><img class="menuNav" src="/img/ic_menu.svg"></li>
-            
+        <body>
+            <header class="head">
+                <nav>
+                    <div class="menu-bar">
+                        <div class="line1"></div>
+                        <div class="line2"></div>
+                        <div class="line3"></div>
+                    </div>
+                </nav>
+            </header>
+            <section class="main-section">
+                <h2 class="name">Besufekad Alemu</h2>
+                <p class="info">Hello! I am a software developer! I can help you build a product, feature or website. Taka a
+                    look of my works. If you like what you see and have a project you need coded, don't hesitate and contact me.</p>
+                <button class="btn">Start collaboration <i
+                        class="fa fa-light fa-arrow-down-to-bracket"></i></button>
+            </section>
+        <div class="vec-image"></div>;
+        <!-- Second Section of the page -->
 
-        </ul>
-        </nav>
-        </header>
-            
-
-          
-  </body>
-  </html>
+        <section class="project">
+            <h1 class="title">
+                Projects
+            </h1>
```

Comment:
- [ ] Your project is perfect, only one thing that I see can be better.  See the image below, you will notice that the font size of the `project` text is small on your design than that of the Figma design. Also the font- type is not the same. I also suggest you use any other `hx` tag (e.g h2, h3.) instead of `h1` for SEO optimization reasons. Kindly fix all these.

Your project | Figma design
---- | ----
![image](https://user-images.githubusercontent.com/84156714/198849615-0f83c31e-f3f1-40e5-8544-b46b0d548a6d.png) | ![image](https://user-images.githubusercontent.com/84156714/198849640-52de79e5-8794-4394-a4c7-cc702e691b14.png)